packages/ui/resolution/tests/resolution-specs.js

```diff
@@ -62,10 +243,10 @@ describe('Resolution Specs', () => {
 	);
 
 	test('should scale pixel measurements for the current screen', () => {
-		const expectedFHD = 24 / 3;
+		const expectedFHD = 24 / 6;
```

Comment:
Could you please explain this change?

Reply:
this is because i change base screen type.